docs/vgclog.md

```diff
@@ -0,0 +1,423 @@
+<!--
+* Copyright (c) 2017, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Verbose Garbage Collection Logs 
+
+[Garbage collection](gc.md)(GC) reclaims used memory in the heap for reuse. During cleanup of the heap, the verbose garbage collection logs capture information about the different garbage collection operations that are involved in the garbage collection cycles. GC operations include GC initialization, stop-the-world processing, finalization, reference processing, and allocation failures. 
+
+A garbage collection event is triggered in the following situations: 
+
+- The VM calls \system.gc() to run a global *stop-the-world* GC. 
+
+- A memory allocation failure. Allocation failures are triggered when a request to allocate an object fails due to a full region of the heap. 
+
+- Available memory in a region of the heap size decreased to a threshold value that triggers a preemptive cleaning of the heap.
+
+- Intermediate or final stages of a concurrent GC cycle are reached. A *stop-the-world* collection is run to complete the incremental or final stage. 
```

Comment:
Intermediate or final increment of a concurrent GC cycle are reached. Intermediate increments could be performed concurrently or STW, but the final one is always STW.

Reply:
Bullets are being removed since this information is now being covered by the Garbage Collection topic.